MultiTarget/PackageReferences/WinAppSdk.props

```diff
@@ -1,10 +1,10 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240829007" />
-    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
-    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2792.45" PrivateAssets="all" />
+    <PackageReference Condition="'$(HasWinUI)' == 'true'" Include="Microsoft.WindowsAppSDK" Version="1.6.240829007" />
+    <PackageReference Condition="'$(HasWinUI)' == 'true'" Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
+    <PackageReference Condition="'$(HasWinUI)' == 'true'" Include="Microsoft.Web.WebView2" Version="1.0.2792.45" PrivateAssets="all" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.1.1" />
+    <PackageReference Condition="'$(HasWinUI)' == 'true'" Include="Microsoft.Windows.CsWinRT" Version="2.1.1" />
```

Comment:
This should never be referenced directly, provided the Toolkit is building with .NET 8.0.403 SDK or greater. Same for "WindowsSdkPackageVersion", we should not be setting that manually anymore if we use that SDK or greater.

Reply:
We also target .NET 6 and 7 for the WindowsAppSDK.

Reply:
That has nothing to do with it. The CsWinRT analyzers and .dll are inside the runtime pack for the Windows SDK.

Reply:
Great, so we can remove this now that we aren't using the `PrivateAssets="All"` workaround from before. Still, this PR is just adding conditions, so we don't include wasdk/winui3 assets under the new Notifications component over in Labs.

Reply:
```suggestion
```

Reply:
Yeah it's fine, let's handle this too in a separate follow up PR 🙂